index.d.ts

```diff
@@ -28,6 +28,13 @@ declare namespace captureWebsite {
 	}
 
 	interface Options {
+		/**
+		Image type.
```

Comment:
Incorrect

Can you also add an example code block of using the `html` option? (Readme too)